### DISCUSSION TITLE
Allow loading models from Stream

### DISCUSSION THREAD

**Discussion by JustBrandonLim** [emgucv/emgucv#941] [Ideas]
Hi, I am currently working on a WPF app that has a `.onnx` model that is embedded as a resource, which the file stream can be obtained. However, I realized that EmguCV only allows loading from file path, instead of a stream.

**Comment by emgucv** [comment]
Thanks for poiting that out. 

Just added this ReadNetFromOnnx function overload to read model from byte array:

1669c216b323e853090a0cd8ca447a03490e733e

It will be included in the next release. 

You should be able to convert a stream to a byte array and load a DNN model from there.

**Reply by JustBrandonLim** [reply]
Thank you so much! Does next release mean the next release of OpenCV? Like 4.9.1?

**Comment by emgucv** [comment]
The next Open CV release should be 4.10

If you need this now you will need to compile from source code:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Building_from_Git
